packages/ember-routing-views/lib/views/link.js

```diff
@@ -216,6 +201,8 @@ var LinkView = EmberComponent.extend({
     this.on(eventName, this, this._invoke);
   },
 
+  _routing: inject.service('-routing'),
```

Comment:
In practice, I think of services as being like specialized controllers. Before services, people would have things like `App.AuthenticationController`, then use `needs` to access it where they needed it. In almost every example I've seen, it's aliased as `[noun]` rather than `[noun]Controller`.

```js
App.UserController = Ember.Controller.extend({
  needs: ['authentication'],

  authentication: alias('controllers.authentication')
});
```

Additionally, in the most recent Ember Data, the store has been refactored to be a service. In order to be consistent with your proposal, we should change the injected property name to `storeService` rather than `store`, but this case doesn't seem to buy us much clarity.

I'd rather us be consistent with service naming across the spectrum, and I think, if considered holistically, not suffixing services with `Service` is consistent with how we typically name things.

(I'm open to argument here, would especially love to hear from others; just wanted to jot down my thought process.)